data/zilliqa/assets/zil13fum43ax8qeprt5s9u6wsmrtw2vsvdrdhmvtrm/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "The Soulless Citadel",
+  "description": "5555 Light Walker Clan cursed to purgatory in The Soulless Citadel. Who can save their souls? We shall see...",
+  "symbol": "TSC",
+  "web": "https://soullesscitadel.com/",
+  "email": "hello@soullesscitadel.com",
+  "publicTeam": true,
+  "product": true,
```

Comment:
remove

Reply:
@balthazar could you please let me know if this would be acceptable?

 - `Usable product  on mainnet`: the largest NFT project on Zilliqa by far (7.3M ZIL primary sale completed in 7 days, 1.47M secondary sale in 14 days in Arky),
 - `with decent activity/users`: with weekly 1k website visitors *after* mint period, 3.5k Discord users,
 - clear token utility: [detailed roadmap](https://soullesscitadel.com/roadmap) bringing secondary marketplace and a strategy game among other things,
 -  token activity: 643 holders of 5555 NFTs with around 20+ token (secondary) sales per day.